packages/client/src/App.tsx

```diff
@@ -27,6 +26,12 @@ axios.interceptors.response.use(
             }
         }
 
+        // Exception for returning the errorMessage provided via the API if available.
+        const apiMessage = error.response?.data?.errorMessage;
```

Comment:
looks like the check above this on line 18 is similar to part of this, would it make more sense to add it into that if/else ladder?

Reply:
If anything, I feel this would be better grouped in an if/else with the statements below (36 & 40) since they are also using logic to determine what message should be returned in the error. But, since the previous dev had the if statement separated with a comment to explain it, I followed that pattern in my code here.